app/views/funding/services/show.html.haml

```diff
@@ -18,14 +18,17 @@
 -# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR~
 -# TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.~
 
-.panel.panel-default
-  .panel-heading.justify-content-between
-    %h4.panel-title
-      = t(:funding)[:download][:header]
-    = link_to "/funding?back=true", title: "Back to funding opportunities" do
-      %span.glyphicon.glyphicon-arrow-left{aria: {hidden: "true"}}
-      = t(:actions)[:back]
-  .panel-body
+
+%header.d-flex.mb-2.default
+  %h2.p-2.mr-auto
+    = t('funding.download.header')
+  %h4.p-2
+    = link_to "/funding?back=true" do
+      = icon('fas', 'arrow-left mr-2')
+      = t('funding.download.back')
```

Comment:
When using content inside a `link_to ... do` like this HAML doesn't trim whitespace which causes extra spacing between elements. You can fix that by doing

```
- succeed <text> do
  = <content before text>
```